iphone/Maps/LocalizedStrings/uk.lproj/Localizable.stringsdict

```diff
@@ -19,11 +16,8 @@
 			<string>%d мітка</string>
 			<key>few</key>
 			<string>%d мітки</string>
-			<key>other</key>
-			<string>%d міток</string>
```

Comment:
No more "other"?

Reply:
@dwaynebailey, I recall that we discussed this particular case, but I don't quite remember the conclusion.

Reply:
https://github.com/organicmaps/organicmaps/pull/10321

Reply:
https://github.com/organicmaps/organicmaps/pull/10321